src/driver/driver.rs

```diff
@@ -17,4 +17,14 @@ extern crate rustdoc as this;
 #[cfg(rustc)]
 extern crate rustc_driver as this;
 
+#[cfg_attr(target_os = "nacl", main_link_name = "nacl_main")]
 fn main() { this::main() }
+
+#[cfg(target_os = "nacl")]
+#[link(name = "ppapi_cpp", kind = "static")]
+#[link(name = "ppapi_simple_cpp", kind = "static")]
+#[link(name = "ppapi_stub", kind = "static")]
+#[link(name = "cli_main", kind = "static")]
+#[link(name = "tar", kind = "static")]
+#[link(name = "nacl_spawn", kind = "static")]
```

Comment:
They are necessary if you'd like to spawn and manage child processes. Which `rustc` does; `rustdoc` may not currently, but there's no harm in adding the libs in case `rustdoc` starts spawning processes at some future date. They also capture stdout/stderr for piping etc, setup env vars (like what a you'd expect on a regular platform), and provide program argument access (in the same way as one would expect for a normal platform).

They should not be default. It would introduce a false dep on all those libs for pure PPAPI plugins (like those using `rust-ppapi`) which should only depend on the C PPAPI.